pkg/controllers/netpol/utils.go

```diff
@@ -142,3 +142,43 @@ func getPodIPv4Address(pod podInfo) (string, error) {
 	}
 	return "", fmt.Errorf("pod %s has no IPv4Address", pod.name)
 }
+
+// fitIn48Charat creates substrings for name or namespace if both combined have more than 48 characters
+func fitIn48Charat(namespace string, name string) (string, string) {
```

Comment:
The name here seems a little odd. Maybe also have it return a single truncated-and-joined string?
```suggestion
// safeJoin joins the namespace and name, ensuring that the result is less than or equal to 48 characters.
func safeJoin(namespace, name string) (string) {
```

Reply:
:+1: